### DISCUSSION TITLE
CreatePage call

### DISCUSSION THREAD

**Discussion by Gwenevere** [gatsbyjs/gatsby#37535] [Help]
Hello, I would like to know if it is possible to get notified when a createPage call fails and still continue with the other pages

**Comment by violy** [comment]
Hello. What is the _fail_ ? an error/warning occurred ? Should you provide more context about your case ?

**Reply by Gwenevere** [reply]
Hey, thank you for your quick answer :) It's a general question, for example, if a component throws an error or accessing a property that is null, or whatever else might fail when creating a page. Is this clear enough?

**Reply by violy** [reply]
the `createPage` calls are inside the[ _bootstrap sequence_ of Gatsby](https://www.gatsbyjs.com/docs/conceptual/gatsby-lifecycle-apis/#bootstrap-sequence) — and then, no Components is mount during this sequence. 

So `createPage` only register a page, with context datas, page component path, page path, etc. but the component is mount after the bootstrap sequence, during the _Build sequence_ — in which errors and warnings is thrown.
